pybossa/data_access.py

```diff
@@ -247,5 +247,7 @@ def valid_user_type_based_data_access(user_type, access_levels):
 @when_data_access()
 def set_default_amp_store(project):
     annotation_config = project['info'].get('annotation_config', {})
-    annotation_config['amp_store'] = True
+    input_data = project['info'].get('input_data') or ''
+    output_data = project['info'].get('output_data') or ''
+    annotation_config['amp_store'] = ('L1' in str(input_data) + str(output_data) == False)
```

Comment:
amp_store to be set to True for all cases when project.info.data_access when not 'L1'

Reply:
Could we consider re-factoring this line to be more clear in what value is being assigned to `annotation_config['amp_store']`? The one-liner is a little confusing at first glance.

For example:

```python
# Set amp_store default to False if 'L1' is in input or output data classifications.
if 'L1' in (str(input_data) + str(output_data)):
  annotation_config['amp_store'] = False
else:
  annotation_config['amp_store'] = True
```